doc/src/ForHosts.dox

```diff
@@ -35,7 +35,8 @@
  *   @ref entity_reference. These are URIs that uniquely identify an
  *   @ref entity within the management system. The form of an entity
  *   reference is entirely determined by the manager. They are
- *   considered to be opaque to the host.
+ *   considered to be opaque handles to the host, even if they look like
```

Comment:
[consider][minor]

```suggestion
 *   considered to be opaque handles in the host, even if they look like
```
or
```suggestion
 *   considered to be opaque handles by the host, even if they look like
```
or
> The host considers them to be opaque handles, even if they look like

Reply:
Just saw that `by` is used in a similar sentence in _Notes for Asset System Integrators_